src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -301,54 +305,57 @@ export default function AuthenticatedHeader({ account, openSettings }: { account
             <LoadingBubble height="16px" width="100px" margin="4px 0 20px 0" />
           </Column>
         )}
-        {!shouldDisableNFTRoutes && (
+        <RowBetween gap="sm">
```

Comment:
@callil this diff scares me did, did you add RowBetween or maybe theres a merge issue?